openlibrary/macros/LoanStatus.html

```diff
@@ -36,29 +36,31 @@
 
 $# Checks to see if patron has actively loan / waitlist for this book
 $ get_loan_for_start_time = time()
-$ user_loan = check_loan_status and ocaid and ctx.user and ctx.user.get_loan_for(ocaid)
+$ user_loan = doc.get('loan') or (check_loan_status and ocaid and ctx.user and ctx.user.get_loan_for(ocaid))
 $ get_loan_for_total_time = time() - get_loan_for_start_time
 
 $ is_edition = doc.key.split('/')[1] == 'books'
 
 $if user_loan:
   $:macros.ReadButton(ocaid, loan=user_loan, listen=listen)
-  $ return_url = doc.url().rsplit('/', 1)[0] + '/do_return/borrow'
-  <form action="$return_url" method="post" class="waitinglist-form return-book">
-    <input type="hidden" name="action" value="return" />
-    <input type="submit" value="$_('Return eBook')" class="cta-btn cta-btn--available" id="return_ebook"/>
-  </form>
-
-  $if render_once('LoanStatus:return-book-js'):
-    <script type="text/javascript">
-      window.q.push(function() {
-        \$('.return-book').on('submit', function(event) {
-          if (!confirm("$_('Really return this book?')")) {
-            event.preventDefault();
-          }
+  $:macros.FormatExpiry(user_loan['expiry'])
+  $if secondary_action:
+    $ return_url = doc.url().rsplit('/', 1)[0] + '/do_return/borrow'
+    <form action="$return_url" method="post"
+          class="waitinglist-form return-book">
+      <input type="hidden" name="action" value="return" />
+      <input type="submit" value="$_('Return eBook')" class="cta-btn cta-btn--vanilla" id="return_ebook"/>
```

Comment:
"Return eBook" grey button border makes it look like it's disabled.  Compare with the disabled check-ins submit button:
![mybooks_return_btn](https://user-images.githubusercontent.com/28732543/208755811-d5c0e4d7-ba8c-4eda-8bfd-7a0b370dc916.png)

![mybooks_disabled_btn](https://user-images.githubusercontent.com/28732543/208755879-a32b05dc-7736-40b0-bc3f-5468d974171b.png)

Maybe a blue border would be better?

Reply:
I agree though this may be better as a separate PR because this style is consistent with what is currently on /account/loans